lib/web/app/handler.go

```diff
@@ -257,6 +257,30 @@ func (h *Handler) HandleConnection(ctx context.Context, clientConn net.Conn) err
 	return nil
 }
 
+// PreflightConnection establishes a connection to a AppServer that can handle
+// application requests. Can be used to ensure the proxy can handle application
+// requests before they arrive.
+func (h *Handler) PreflightConnection(ctx context.Context, identity *tlsca.Identity) error {
```

Comment:
Can we rename it to more meaningful like `checkIfAppIsAccesable` ?  At first glance the `PreflightConnection` is quite enigmatic and without reading the comment hard to grasp what it does.

Reply:
Do you mean renaming all occurrences of "preflight connection" or just this one?

Reply:
Yes I was thinking about renaming it because  as far I understand the `Preflight" mostly referee to HTTP preflight request: 

> A preflight request is a small request that is sent by the browser before the actual request. It contains information like which HTTP method is used, as well as if any custom HTTP headers are present. The preflight gives the server a chance to examine what the actual request will look like before it’s made.

and here we are just doing the tcp health check in order to check if any app agent is available. 

but I don't have strong opinion about it.